pkg/controller/expcertificates/issuing/issuing_controller.go

```diff
@@ -200,22 +229,46 @@ func (c *controller) ProcessItem(ctx context.Context, key string) error {
 		return nil
 	}
 
-	switch cond.Reason {
 	// If the certificate request has failed, set the last failure time to now,
 	// and set the Issuing status condition to False with reason.
-	case cmapi.CertificateRequestReasonFailed:
+	if cond.Reason == cmapi.CertificateRequestReasonFailed {
 		return c.failIssueCertificate(ctx, log, crt, req)
+	}
 
-		// If the CertificateRequest is valid, verify its status and update
-		// accordingly.
-	case cmapi.CertificateRequestReasonIssued:
-		return c.issueCertificate(ctx, log, nextRevision, crt, req)
+	// Verify the CSR options match what is requested in certificate.spec.
+	// If there are violations in the spec, then the requestmanager will handle this.
+	requestViolations, err := certificates.RequestMatchesSpec(req, crt.Spec)
+	if err != nil {
+		return err
+	}
+	if len(requestViolations) > 0 {
+		log.Info("CertificateRequest does not match Certificate")
```

Comment:
Can you expand this to add that it is waiting on the keymanager similar to other logs?